src/styles/FontUploader.css

```diff
@@ -4,12 +4,25 @@
     border: 2px dashed var(--color-border);
     display: flex;
     flex-direction: column;
-    padding: 2rem 2.5rem;
+    padding: 1.5rem 2rem;
     text-align: center;
     width: 100%;
+    height: 7.5em;
```

Comment:
Might suggest using whatever the equivalent value in `rem` is.

If I’m not mistaken, font size on this element is the same as root, so might even be `7.5rem`.